frame/staking-rewards/src/lib.rs

```diff
@@ -289,10 +333,29 @@ pub mod pallet {
 		#[transactional]
 		fn split(
 			_who: &Self::AccountId,
-			_position: &Self::PositionId,
-			_ratio: Permill,
+			position: &Self::PositionId,
+			ratio: Permill,
 		) -> Result<[Self::PositionId; 2], DispatchError> {
-			Err("Not implemented".into())
-		}
+            let old_position = Stakes::<T>::try_mutate(position, |old_stake| {
```

Comment:
reductions for each rewarded asset type needs still to be handled here

Reply:
Yes I am aware, I was not able understand how it will be used so waiting for other PR to get better understanding.